src/main/resources/org/jenkinsci/plugins/github_branch_source/BranchDiscoveryTrait/help-strategyId.html

```diff
@@ -8,8 +8,8 @@
         </dd>
         <dt>Only branches that are also filed as PRs</dt>
         <dd>
-            This option exists to preserve legacy behaviour when upgrading from older versions of the plugin.
-            NOTE: If you have an actual use case for this option please file a pull request against this text.
+            Similar to discovering pull requests directly, but env.GIT_BRANCH is set to the branch name rather than PR-#,
+            and github pull-request checks are not updated.
```

Comment:
I think it would be better to reword as something like “the status notification for the build will only be applied to the commit, the pull requests status will not be updated” (my text needs revision, but should point you in the direction more clearly